src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,651 @@
+
+
+def remove_existing_experience_data(bind):
+    """
+    We remove any existing PrivacyExperience and PrivacyExperienceConfig records,
+    to allow for a "clean-slate" in creation of _new_ PrivacyExperienceConfigs based on the
+    "out of the box" templates.
+
+    This means that any existing user-edited data on PrivacyExperienceConfig records will need to be
+    _manually_ re-added to the post-migration state. It will not be migrated as part of the automatic migration.
+    """
+
+    def delete_experiences(bind):
+        """
+        Delete all PrivacyExperience records.
+
+        These records will be effectively re-created as a result of the "out of the box" creation
+        of 'new-style' PrivacyExperienceConfig records post-migration.
+        """
+        bind.execute(
+            text(
+                """
+                DELETE FROM privacyexperience;
+                """
+            )
+        )
+
+    def delete_experience_configs(bind):
+        """
+        Delete all PrivacyExperienceConfig records.
+
+        These records will be effectively re-created via the server's "out of the box" creation
+        of the updated PrivacyExperienceConfig records, post-migration.
+        """
+        bind.execute(
+            text(
+                """
+                DELETE FROM privacyexperienceconfig;
+                """
+            )
+        )
+
+    delete_experiences(bind)
+    delete_experience_configs(bind)
+
+
+def determine_needed_experience_configs(bind):
+    """
+    Our data model changes necessitate the creation of new Experience Configs. Now that "where" a Notice is served
+    is being moved from the "Notice" constructs to "Experiences", existing Experience Configs are not sufficient
+    to capture the full range of configuration on current Notices.
+    """
+
+    notice_query = """select id, name, regions, displayed_in_overlay, displayed_in_privacy_center, notice_key from privacynotice;"""
+    experience_config_query = """SELECT id, component, disabled, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title, banner_description, banner_title, reject_button_label, acknowledge_button_label from privacyexperienceconfig;"""
+
+    existing_notices: ResultProxy = bind.execute(text(notice_query))
+    for existing_notice in existing_notices:
+        notice_key = existing_notice["notice_key"]
+        component_mapping = NOTICES_TO_EXPERIENCE_CONFIG.get(notice_key)
+        if component_mapping:
+            regions_to_add = set(existing_notice["regions"]).difference(
+                REGIONS_TO_REMOVE
+            )
+            if existing_notice["displayed_in_overlay"]:
+                experience_config = component_mapping.get(ComponentType.Overlay)
+                reconciled_experience_config_map[experience_config]["regions"].update(
+                    regions_to_add
+                )
```

Comment:
yup great callout! that seems _generally_ true from the data sample we've seen so far, but there does seem to be at least one exception. i'd noted this as a "manual step" in the draft of my [release plan](https://ethyca.atlassian.net/wiki/spaces/EN/pages/2952298561/DRAFT+Multilang+release+plan+2.32.0#Manual-steps) (language is admittedly a bit vague there), i'll clarify more precisely what we need to look for and how it should be addressed.

even though we know this will lead to some necessary manual adjustments in a few cases, i still think the tradeoff we gain in simplicity/predictability is worth it!